.travis.yml

```diff
@@ -4,7 +4,7 @@ env:
     - TOXENV=py27
     - TOXENV=py34
 install:
-    - pip install -r requirements.txt coveralls flake8 sphinx testimony tox
+    - pip install -r requirements-freeze.txt coveralls flake8 sphinx testimony tox
```

Comment:
I think all other packages can be removed since they should be included in `requirements-freeze.txt`?